meetings/2020-04-14.md

```diff
@@ -5,21 +5,63 @@
 ## [Meeting notes](https://docs.google.com/document/d/1hrT3HvHiQIP5jKiHsaeOLq2IRIdUKkM3ipbVXUd9ci8/edit?usp=sharing)
```

Comment:
I'm also going to go back and set the meeting google doc as read-only after each meeting so that it's "archived" as it was written.